tutorials/llm/llama-3/pruning-distillation/03_b_width_pruning.ipynb

```diff
@@ -0,0 +1,92 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8bc99d2f-9ac6-40c2-b072-12b6cb7b9aca",
+   "metadata": {},
+   "source": [
+    "### Step 3: Prune the finetuned-teacher model to create a student\n",
+    "In the second method, we will width-prune. In width-pruning, we trim the neurons, attention heads and embedding channels. \n",
+    "\n",
+    "Refer to the ``NOTE`` in the **_step-by-step instructions_** section of [introduction.ipynb](./introduction.ipynb) to decide which pruning techniques you would like to explore."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9207ed14-2f37-4712-88f3-543a128663ac",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### Step 3.b.: Using width-pruning\n",
+    "To width-prune the model, we do the following:\n",
+    "- prune (trim) the MLP intermediate dimension from 14336 to 9216.\n",
+    "- prune the hidden size from 4096 to 3072.\n",
+    "- and retrain the attention headcount and number of layers\n",
+    "\n",
+    "For width-pruning we will use the [megatron_gpt_prune.py](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_prune.py) script in the NeMo Framework. To see the detailed list of parameters for width-pruning, you can view the [megatron_gpt_prune.yaml](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/conf/megatron_gpt_prune.yaml) file.\n",
```

Comment:
fix punctuation

"For width-pruning, we will use the [megatron_gpt_prune.py](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_prune.py) script in the NeMo Framework. To see the detailed list of parameters for width-pruning, you can view the [megatron_gpt_prune.yaml](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/conf/megatron_gpt_prune.yaml) file.\n",